example/angular_components_example/build.release.yaml

```diff
@@ -0,0 +1,24 @@
+targets:
+  $default:
+    builders:
+      angular|angular:
+        enabled: True
+      angular_gallery|angular_gallery:
+        options:
+          styleUrls: ["gallery.scss.css"]
+          galleryTitle: "AngularDart Gallery"
+          examples: "app_layout_example,material_button_example,material_checkbox_example,material_chips_example,material_dialog_example,material_expansionpanel_example,material_icon_example,material_input_example,material_list_example,material_menu_example,material_popup_example,material_progress_example,material_radio_example,material_select_example,material_spinner_example,material_tab_example,material_toggle_example,material_tooltip_example,material_tree_example,material_yes_no_buttons_example,scorecard_example"
```

Comment:
Could you use a yaml array here? This is tough to parse...

Reply:
TLDR; not at this time.

This is an ugly artifact from part of the internal build process. I'm trying to figure out a way to improve the readability here.